sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -869,15 +869,18 @@ lsc_gather(AccessorTy acc, __ESIMD_NS::simd<uint32_t, N> offsets,
 /// for the data size are \c lsc_data_size::u32, \c lsc_data_size::u64,
 /// \c lsc_data_size::u8, \c lsc_data_size::u16.
 /// When data size is either  \c lsc_data_size::u8 or \c lsc_data_size::u16
-/// the data is treated as 32 bit data. Allowed NElts values for 64 bit
-/// data are 1, 2, 3, 4, 8, 16, 32, 64.
+/// the data is treated as 32 bit data. Allowed \c NElts values for 64 bit
+/// data are 1, 2, 3, 4, 8, 16, 32, 64. Allowed \c NElts values for 32 bit
+/// data are 1, 2, 3, 4, 8, 16, 32, 64, 128. Allowed \c NElts values for 16 bit
+/// data are 2, 4, 8, 16, 32, 64, 128, 256. Allowed \c NElts values for 8 bit
+/// data are 4, 8, 12, 16, 32, 64, 128, 256, 512.
 /// When NElts is greater than 64 and the data size is not \c
```

Comment:
This sentence seems not correct. The load of 128 shorts, or 256-bytes doesn't require 8 byte alignment because it is treated as dwords, right?

Reply:
Right, the actual restriction is on the final message's vector size. For 128 short elements, they can still fit in 64 32-bit channels and therefore no need to force 8-byte alignment.